### PR TITLE
OTC-276: enabling searching not only matched payments etc

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openimis.atlassian.net/browse/OTC-276

changes:
- adding new resources for payment statuses and "RejectedReason" label. 
- enabling looking for another payments, not only paid etc
otc-276
otc-276c

### CODE REVIEW

**Comment by delcroip** [comment]
@dragos-dobre  add status for cancelled payment

File: IMIS_BL/General.vb [hunk 1]
```diff
@@ -1006,25 +1006,21 @@ Public Class GeneralBL
...
-                'dtStatus.Rows.Add(-3, getMessage("T_FAILEDSTATUS"))
-                'dtStatus.Rows.Add(1, getMessage("T_PAYMENTREQUESTED"))
-                'dtStatus.Rows.Add(2, getMessage("T_PAYMENTREQUESTED"))
-                'dtStatus.Rows.Add(3, getMessage("T_PAYMENTREQUESTED"))
-                'dtStatus.Rows.Add(4, getMessage("T_PAYMENTRPAID"))
-                'dtStatus.Rows.Add(5, getMessage("T_PAYMENTMATCHED"))
-
-                dtStatus.Rows.Add(5, getMessage("T_PAYMENTMATCHED"))
-                dtStatus.Rows.Add(4, getMessage("T_UNMATCHED"))
+                dtStatus.Rows.Add(-3, getMessage("T_FAILEDSTATUS"))
```
Comment (dragos-dobre) [inline IMIS_BL/General.vb:1009]: Please create a constant list for the PaymentStatus and use it here. Will be easier to understand.

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: IMIS_BL/General.vb (hunk 1 above)
Comment (sniedzielski) [reply IMIS_BL/General.vb:1009]: ok

**Comment by sniedzielski** [comment]
@dragos-dobre I added static dict with PaymentStatus. I also fixed conflict with OTC-10 merged into develop. You can re-check this PR.

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**